init_mongo.js

```diff
@@ -80,8 +80,10 @@ db.createUser({ user: "core",
   ]
 });
 db.createCollection("event");
+db.reading.createIndex({"device": 1}, {unique: false});
 db.createCollection("reading");
 db.createCollection("valueDescriptor");
+db.reading.createIndex({"device": 1}, {unique: false});
```

Comment:
I want to get confirmation from @AnthonyMBonafide that this same line should be added twice.

Reply:
Fixed the copy/paste issue